src/libs/elektra/key.c

```diff
@@ -73,6 +73,21 @@
  * ordering or different Models of your configuration.
  */
 
+/*
+ * @internal
+ *
+ * Allocates and initializes a key
+ * @returns 0 if allocation did not work, the key otherwise
+ */
+static Key * elektraKeyMalloc (void)
+{
+	Key * key = (Key *) elektraMalloc (sizeof (Key));
```

Comment:
This function could be a one-liner: `return elektraCalloc (sizeof (Key));`.
This will initialize all members to 0, making it a bit more flexible for future changes (e.g. adding another pointer member). Default values different from 0 of course have to be initialized correctly, but it's less common.

There's also no need to cast `void *` to other pointer types, I think.